sdk/tests/deqp/functional/gles3/es3fFboRenderTest.js

```diff
@@ -704,7 +704,48 @@ goog.scope(function() {
         );
     };
 
+    /**
+     * deinit
+     */
     es3fFboRenderTest.FboRenderCase.prototype.deinit = function() {
```

Comment:
There are many errors caused by uncleared states set by former executed test cases. Here I added codes to clear gl states.
Referenced from: 
https://github.com/KhronosGroup/WebGL/blob/master/sdk/tests/deqp/functional/gles3/es3fMultisampleTests.js#L216